components/mixins/profile/profile-personal-tax-profile-details.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
```

Comment:
Can this be an object under personal finances? So, personalFinances.taxProfile.fields? That way it'll all be under the personal finances folder, but neatly in a tax object.